draft-ietf-core-oscore-groupcomm.md

```diff
@@ -324,6 +333,10 @@ The new parameter Group Manager Authentication Credential specifies the authenti
 
 The new parameter Group Encryption Algorithm identifies the algorithm to use for encryption and decryption, when messages are protected in group mode (see {{mess-processing}}). This algorithm MAY provide integrity protection. If this parameter is not set, the group mode is not used in the group.
 
+The following non-authenticated algorithms can be used as Group Encryption Algorithm: A128CBC, A192CBC, and A256CBC {{RFC9459}}. The non-authenticated algorithm ChaCha20 {{ChaCha}} is also suitable to consider, although using it will first require its registration in the "COSE Algorithms" Registry.
+
+The following non-authenticated algorithms MUST NOT be used as Group Encryption Algorithm: A128CTR, A192CTR, and A256CTR {{RFC9459}}.
```

Comment:
the problem with a list like this is that it's never going to be exhaustive... can we skip it? Christian only asked to add which parameter are particularly suitable...

Reply:
Addressed at https://github.com/core-wg/oscore-groupcomm/commit/d2492130224e71a209eff6f899be346b816889de